src/dependencies.jl

```diff
@@ -106,7 +106,7 @@ end
 can_use(::Type{AptGet}) = has_apt && OS_NAME == :Linux
 package_available(p::AptGet) = can_use(AptGet) && !isempty(available_versions(p))
 function available_versions(p::AptGet)
-    vers = Compat.ASCIIString[]
+    vers = Compat.String[]
```

Comment:
Also since [`String` is imported](https://github.com/JuliaLang/BinDeps.jl/blob/1e03521858e51ea2fa681fbdf75bd38bbcf1d092/src/BinDeps.jl#L4) this should be `String` and not `Compat.String`.